chat-widget/src/components/webchatcontainerstateful/WebChatContainerStateful.tsx

```diff
@@ -26,13 +27,15 @@ export const WebChatContainerStateful = (props: IWebChatContainerStatefulProps)
     
     const localizedTexts = {
         ...defaultMiddlewareLocalizedTexts,
+        ...defaultChatButtonLocalizedTexts,
```

Comment:
Please don't add complexities in WebChatContainer. The localized texts for Chat Button should come as controlProps for the ChatButton component itself